### DISCUSSION TITLE
Change yes/no to true/false

### DISCUSSION THREAD

**Discussion by remcohaszing** [publiccodeyml/publiccode.yml#115] [General]
This specification defines a YAML file which should use `yes`/`no` is accepted in several places. However, the meaning of `yes`/`no` is ambiguous in YAML. In YAML 1.1 and older [these values represent booleans](https://yaml.org/type/bool.html), but in YAML 1.2 [they don’t](https://yaml.org/spec/1.2.0/#id2602744), meaning they represent strings.

I suggest to remove this ambiguity by changing them to unambigous booleans (`true`/`false`) in the specification.

**Comment by ruphy** [comment]
thank you @remcohaszing, I think it's a good idea

**Comment by libremente** [comment]
This issue was related https://github.com/publiccodeyml/publiccode.yml/issues/8

**Comment by libremente** [comment]
See also https://github.com/italia/publiccode-editor/issues/48

**Comment by remcohaszing** [comment]
[`yaml-language-server` 1.0](https://github.com/redhat-developer/yaml-language-server/blob/main/CHANGELOG.md#100) and [VSCode Yaml 1.0](https://github.com/redhat-developer/vscode-yaml/releases/tag/1.0.0) have just been released, where they switched from using YAML 1.1 to YAML 1.2. This means they will now mark the values `yes`/`no` as invalid, which comes from JSON schema validation based the [publiccode JSON schema](https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/publiccode.json) in the schema store.

**Comment by bfabio** [comment]
I think explicitly base the format on YAML 1.2  is the number one issue we need to address. Moving to Discussions.
